css-layout-classes/index.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>CSS Layout Classes</title>
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <div class="column-full">
+          <header>
+            <h2>Choose Your Character!</h2>
+          </header>
+        </div>
+      </div>
+
+      <div class="row">
+        <div class="column-full">
+          <article>
```

Comment:
You can remove these three elements, and just keep the `row` with the `column-one-third` child elements